.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
+
+[*]
+indent_size = 4
+end_of_line = lf
+
+[*.{c,h,cxx,am,m4}]
+indent_style = tab
```

Comment:
couldn't that apply to everything, and only special-case some other stuff?  Or is the goal here not to risk overriding an automatic detection for stuff we don't know about for sure?